test/server.test.js

```diff
@@ -1929,3 +1937,38 @@ test('GH-877 content-type should be case insensitive', function (t) {
     });
     client.end();
 });
+
+test('GH-733 if request is closed early, stop processing', function (t) {
```

Comment:
For this test, you'll need to verify that all of the post processing on the server has completed. e.g. audit logs, timers, etc.